src/App.js

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Teams from './components/teams/Teams';
+import TeamStats from './components/teamStats/TeamStats';
+import Nabar from './components/Navbar/Nabar';
+
+function App() {
+  return (
+    <div className="bg-sky-700 px-12 min-h-screen">
+      <Router>
+        <Nabar />
+        <Routes>
+          <Route path="/" element={<Teams />} />
+          <Route path="/:teamId/statistics" element={<TeamStats />} />
+        </Routes>
+      </Router>
+    </div>
+  );
+}
+
```

Comment:
Nice job so far, Your project looks great👍🏼. Please let's try to make it responsive as it currently looks different from the given design on the mobile screen😉.

Your project
<img width="526" alt="image" src="https://github.com/migo1/nba-teams-stats/assets/64643995/cf41ece2-15e0-439f-b027-ffdcbead3522">

design
<img width="340" alt="image" src="https://github.com/migo1/nba-teams-stats/assets/64643995/44a2de07-b91d-4b77-9e5b-c9dbfe995c94">


Kindly note that this is the same for all pages, as the mobile view wasn't considered😉. Do well to make the necessary changes to make your project match the design👍🏼.